parser/lexer.go

```diff
@@ -330,6 +330,30 @@ func (lt *LexToken) String() string {
 	return fmt.Sprintf("%q (%d) = %T{%v} %d:%d", name, lt.token, lt.value, lt.value, lt.pos.Lineno, lt.pos.ColOffset)
 }
 
+// TokenToName returns the string name of a given token
+func (lt *LexToken) TokenToName() string {
+        name, ok := tokenToString[lt.token]
```

Comment:
perhaps we could make the token implement `fmt.Stringer` instead ?

Reply:
sure, i'll take a look. random question, i hacked in comment support in my branch because I wanted to capture those.  any interest in supporting comment capture in gpython?  I saw that it was tossed away in lexer.go.

Reply:
IMHO, it would be useful.
but @ncw would know best.

Reply:
Comment support sounds interesting. Is it part of the standard Python lexer? I don't remember